Franchise links stats

darkstars31 opened the pull request.

JesseMH commented:
1. Be careful about Dynamic classNames like in line 46 of franchise.tsx:  <div className={`grid grid-cols-${COLUMNS} text-xs`}>

From what I understand, Tailwind does not handle that well, and you might get an intermittent issue like I did with the rating bar colors. From their docs ([Source](https://tailwindcss.com/docs/content-configuration#dynamic-class-names)):
```The most important implication of how Tailwind extracts class names is that it will only find classes that exist as complete unbroken strings in your source files.```
```If you use string interpolation or concatenate partial class names together, Tailwind will not find them and therefore will not generate the corresponding CSS.```

2. Add error handling for FaceIT Profile links. When I checked earlier, a few players on UPS were linking to https://www.faceit.com/en/players/**undefined**

3. Add a unique Key to the team.players.map in the franchise.tsx return section, it's throwing errors(Warning: Each child in a list should have a unique "key" prop.) here:
`{ team.players.map( player => 
     <PlayerRow franchisePlayer={player} team={team} />
)}`

4. The background image should probably be full page, right now it stops about halfway down. 

5. Stats in PlayerRow look clickable but aren't links. Consider only adding pointer events only where needed.

darkstars31 commented:
I think I have a solution for `dynamic classnames` issue. By default webpack does tree shaking to remove unused code from production builds - tailwind included, so it cannot detect which classNames to keep if it's dynamic. However if we include the necessary classNames in safelist it will be included.
```
safelist: [
    { pattern: /grid-cols-./ },
    /* ["animate-[fade-out-right_1s_ease-in-out]", "animate-[fade-out-left_1s_ease-in-out]"] */
  ],
  ```
  Fixed up faceit links being undefined - This probably shouldn't happen so I need to dig into why those are coming up undefined.
  
  Fixed unique key for `team.plaayers`
  Fixed Background image to be full screen
  Removed cursor on hover for entire row and applied it only to Usernames/Discord/Faceit links. Also changed the name to be "blue" so that it looks more like a link.
